src/app/pages/signature-request/components/sign-action.tsx

```diff
@@ -21,10 +21,24 @@ function useSignMessageSoftwareWallet() {
   );
 }
 
-interface SignActionProps {
+export type SignatureMessageType = 'string' | 'structured';
```

Comment:
Should we call `'string'` something else? text? ascii/utf8? I feel like this could be confusing.

Reply:
I replaced 'string' with 'utf8' as that's the default encoding used in our signMessage function (which used Buffer.from())